src/utils/taro/get-system-info.ts

```
@@ -1,34 +1,35 @@
+  return typeof Taro.getWindowInfo === 'function'
+    ? Taro.getWindowInfo()
+    : Taro.getSystemInfoSync()
 }
 
 /**
  * 获取应用基础信息，兼容新旧 API
  * @returns {Taro.getAppBaseInfo.Result} 应用基础信息
  */
 export const getAppBaseInfo = (): Taro.getAppBaseInfo.Result => {
-  return canIUse('getAppBaseInfo') ? taroGetAppBaseInfo() : getSystemInfoSync()
+  return typeof Taro.getAppBaseInfo === 'function' ? Taro.getAppBaseInfo() : Taro.getSystemInfoSync()
```

ESLint

[error] Replace `·?·Taro.getAppBaseInfo()` with `⏎····?·Taro.getAppBaseInfo()⏎···`

(prettier/prettier)